readme.md

```diff
@@ -79,13 +79,17 @@ exports.createPages = ({ graphql, actions }) => {
     graphql(`
       //graphql query
     `).then(result => {
+      const postsPerPage = 5
+      const { edges } = result.data.posts
       createPaginatedPages({
-        edges: result.data.posts.edges,
+        edges: edges,
```

Comment:
Could just do `edges,` here, but thought it'd be more readable for everyone this way